app/editor.js

```diff
@@ -300,6 +301,7 @@ ved.cql.init = function(data) {
   ved.cql.schema = new cql.schema.Schema(fieldSchemas);
   var summary = vg.util.summary(data);
   ved.cql.stats = new cql.stats.Stats(summary);
+  ved.cql.views = new d3.map();
```

Comment:
just use object -- we don't use `d3.map` anywhere else so it's better to be consistent.